### PR TITLE
some prep in direction of using ansible_role_boostrap now that it includes the smartos bits

### PR DESCRIPTION
**Pull request by alexschlessinger**
also some prep for moving ansible_role_percona out (linux)
needs turbotiming etc

### CODE REVIEW

File: hosts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [percona-mysql]
-percona-test.stage.modcloth.com ansible_ssh_host=192.168.113.185 ansible_ssh_user=root ansible_python_interpreter=/opt/local/bin/python 
+percona-56-ecomm-replica.stage.modcloth.com ansible_ssh_host=192.168.113.162 ansible_ssh_user=root ansible_python_interpreter=/opt/local/bin/python
```
Comment (rafecolton) [inline hosts:2]: Hmmm, we've been using the format `<ip> hostname=<hostname>`, but it looks like this is the correct way to do it.  Would recommend setting the `hostname=<hostname>` var or updating the bootstrap role to use `inventory_hostname` (of course the latter would require changes all over the place :crying_cat_face:)

File: roles/ansible-role-percona/tasks/configuration.yml [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
-  template: src=my.cnf.j2 dest=/opt/local/etc/my.cnf owner=mysql group=mysql mode=0644
-  notify:
-  - restart percona
+  template: src=my.cnf.j2 dest=/etc/my.cnf owner=mysql group=mysql mode=0644
 
+- name: symlink config on solaris
+  file: dest=/opt/local/etc/my.cnf src=/etc/my.cnf state=link force=yes
+  when: ansible_os_family == "Solaris"
 
+- name: restart percona
```
Comment (rafecolton) [inline roles/ansible-role-percona/tasks/configuration.yml]: Should this be a handler notified by this task? As opposed to restarting every time the task is task is run

**Comment by meatballhat** [comment]
@alexschlessinger this still valid?

**Comment by alexschlessinger** [comment]
percona-ansible was a repo I was playing with for provisioning smartos
percona boxes before we had ansible bootstrap support for smartos, and
before we were using roles. as such this repo had embedded roles for
smartos bootstrap, nagios, and percona.

the percona stuff is now in ansible-role-percona, and this particular issue
resolved (handler notification). The nagios and bootstrap stuff Rafe copied
into the new bootstrap.

as such this repo is basically defunct, but i had some thoughts to
resurrect it to manage the mysql replica chain as part of the mysql 5.6
upgrade project.

**Comment by meatballhat** [comment]
:clap: :poop: :dancer:

**Pull request closed without merging**